test/rules/bz2006561.yaml

```diff
@@ -0,0 +1,52 @@
+# Tests for https://bugzilla.redhat.com/show_bug.cgi?id=2006561
+# for metric kube_pod_spec_volumes_persistentvolumeclaims_info, make sure duplicated records returned by 'on(namespace, pod)' will not cause error when evaluating "HighlyAvailableWorkloadIncorrectlySpread"
+# For the case of bug 2006561, for kube_pod_spec_volumes_persistentvolumeclaims_info is duplicated with "on(namespace, pod)" aggregation because of multiple uids exists for the same namespace and pod combination.
```

Comment:
```suggestion
# For the case of bug 2006561, kube_pod_spec_volumes_persistentvolumeclaims_info is duplicated with "on(namespace, pod)" aggregation because multiple uids exist for the same namespace and pod combination.
```